packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -812,7 +821,7 @@ export abstract class FluidDataStoreContext
 	 * @deprecated 0.18.Should call request on the runtime directly
 	 */
 	public async request(request: IRequest): Promise<IResponse> {
-		const runtime = await this.realize();
+		const runtime = await this.realizeInternal();
```

Comment:
I'm not sure how to think about the change (difference of calling realize vs. realizeInternal).
Here, for example, I'd think that if the app is calling realize(), then likely we want to do conversion (if required).
Given that was deprecated for long time, maybe (with double-checking) the better strategy is to remove this API.

As for other places - not sure. Should summarizer path avoid conversion? Why?